Documentation/gettingstarted/memcached.rst

```diff
@@ -0,0 +1,323 @@
+- **Alliance-Tracker**: can get any coordinates but not set any.
+
+To keep the setup small, we will launch a small number of pods to represent a larger environment:
+
+- **memcached-server** : A k8s service represented by a single pod running a memcached server (label app=memcd-server).
+- **a-wing** memcached binary client : A pod representing an A-wing starfighter, which can update its coordinates and read it via the binary memcached protocol (label app=a-wing).
+- **x-wing** memcached text-based client : A pod representing an X-wing starfighter, which can update its coordinates and read it via the text-based memcached protocol (label app=x-wing).
+- **alliance-tracker** memcached binary client : A pod representing the Alliance Fleet Tracker, able to read the coordinates of all starfighters (label name=fleet-tracker).
+
+
+Memcached clients access the *memcached-server* on TCP port 11211 and send memcached protocol messages to it.
+
+.. image:: images/cilium_memcd_gsg_topology.png
+
+
+The file ``memcd-sw-app.yaml`` contains a Kubernetes Deployment for each of the pods described
+above, as well as a Kubernetes Service *memcached-server* for the Memcached server.
+
+.. parsed-literal::
+
+    $ kubectl create -f \ |SCM_WEB|\/examples/kubernetes-memcached/memcd-sw-app.yaml
+    deployment.extensions/memcached-server created
+    service/memcached-server created
+    deployment.extensions/a-wing created
+    deployment.extensions/x-wing created
+    deployment.extensions/alliance-tracker created
+
+Kubernetes will deploy the pods and service in the background.
+Running ``kubectl get svc,pods`` will inform you about the progress of the operation.
+Each pod will go through several states until it reaches ``Running`` at which
+point the setup is ready.
+
+.. parsed-literal::
+
+    $ kubectl get svc,pods
+    NAME                       TYPE        CLUSTER-IP   EXTERNAL-IP   PORT(S)     AGE
+    service/kubernetes         ClusterIP   10.96.0.1    <none>        443/TCP     31m
+    service/memcached-server   ClusterIP   None         <none>        11211/TCP   14m
+
+    NAME                                    READY   STATUS    RESTARTS   AGE
+    pod/a-wing-67db8d5fcc-dpwl4             1/1     Running   0          14m
+    pod/alliance-tracker-6b6447bd69-sz5hz   1/1     Running   0          14m
+    pod/memcached-server-bdbfb87cd-8tdh7    1/1     Running   0          14m
+    pod/x-wing-fd5dfb9d9-wrtwn              1/1     Running   0          14m
+
+We suggest having a main terminal window to execute *kubectl* commands and two additional terminal windows dedicated to accessing the **A-Wing** and **Alliance-Tracker**, which use a python library to communicate to the memcached server using the binary protocol.
+
+In **all three** terminal windows, set some handy environment variables for the demo with the following script:
+
+.. parsed-literal::
+
+   $  curl -s \ |SCM_WEB|\/examples/memcd-env.sh | source
```

Comment:
seems like this should be "examples/kubernetes-memcached/memcd-env.sh" instead.  

Also, even with the fixed-up URL, this doesn't work for me on OS X: 

dans-Air:cilium danwent$ curl -s https://raw.githubusercontent.com/cilium/cilium/gsg-memcached/examples/kubernetes-memcached/memcd-env.sh | source
-bash: source: filename argument required
source: usage: source filename [arguments]
(23) Failed writing body

Reply:
Update done.